change config description to avoid ambiguity (#41)
zz-jason/tidb-dev-guide: get-started/setup-an-ide.md

![tidb in goland](https://user-images.githubusercontent.com/30543181/118812264-7fb9cc00-b8e0-11eb-8825-223afa93b2f0.png)

> **Step 4**. `cd <tidb-dir>` and Executing it to adding these three config files to `./.idea/runConfigurations/` directory. 

Reviewer: ```suggestion
**Step 4**. `cd <tidb-dir>` and execute following commands to add these three config files to `./.idea/runConfigurations/` directory. 
```

> **Step 4**. ```cd <tidb-dir>``` and Executing it to adding these three config files to `./.idea/runConfigurations/` directory. 

Reviewer: ```suggestion
**Step 4**. `cd <tidb-dir>` and Executing it to adding these three config files to `./.idea/runConfigurations/` directory. 
```

> **Step 4**. you need to replace ```{{PD_ADDRESS}}``` with your pd address (such as 127.0.0.1:2379) for following scipts. ```cd <tidb-dir>``` and Executing it to adding these three config files to `<tidb-dir>/.idea/runConfigurations/` directory. 

Reviewer: ```suggestion
**Step 4**. you need to replace `{{PD_ADDRESS}}` with your pd address (such as 127.0.0.1:2379) for following scipts. `cd <tidb-dir>` and Executing it to adding these three config files to `<tidb-dir>/.idea/runConfigurations/` directory. 
```

Reviewer: It is a bit duplicated we mention two times about the working directory.

Author: Delete the last one.

</component>
EOF

cat <<EOF > playground_attach_4001.xml
<component name="ProjectRunConfigurationManager">
  <configuration default="false" name="playground attach 4001" type="GoApplicationRunConfiguration" factoryName="Go Application">
    <module name="tidb" />
    <working_directory value="$PROJECT_DIR$" />

>     <parameters value="--path={{PD_ADDRESS}} --store=tikv --status=10081 -P 4001 " />

Reviewer 2: ```suggestion
    <parameters value="--path=127.0.0.1:2379 --store=tikv --status=10081 -P 4001 " />
```

  </configuration>
</component>
EOF
```

> After executing script, you can check whether to have three config in the directory. 
> ```bash

Reviewer: ```suggestion
After executing script, you can check whether to have three config in the directory. 

```bash
```
